styleguide/js/zoolander.js

```diff
@@ -1,3 +1,26 @@
+jQuery(document).ready(function(){
```

Comment:
This will be the third `jQuery(document).ready()` function in `zoolander.js`.  Since the file doesn't really do all that much, it seems it would make more sense to just combine them into a single `jQuery(document).ready()` function.   

Additionally, make sure the function declaration has the `$` variable declared, if you plan on using `$` instead of `jQuery`. 
```
jQuery(document).ready(function ($) {
```

After doing this, any nested `jQuery()` functions could probably just be replaced with `$` to make code a little more succinct.